test/parameter_test.py

```diff
@@ -678,6 +702,14 @@ def complete(self):
         self.assertTrue(self.run_locally_split('mynamespace.A --mynamespace.A-p 200 --expected 200'))
         self.assertFalse(self.run_locally_split('mynamespace.A --A-p 200 --expected 200'))
```

Comment:
I can get the two assertTrue statements to work for ListParameter, but the `self.assertFalse(self.run_locally_split('mynamespace.A --A-p [1,2,3] --expected [1,2,3]'))` returns `SystemExit: 2` , ending the stacktrace with argparse.py causing the exit. statement.